src/platform/graphics/webgpu/webgpu-graphics-device.js

```diff
@@ -817,6 +823,104 @@ class WebgpuGraphicsDevice extends GraphicsDevice {
         }
     }
 
+    /**
+     * Clear the content of a storage buffer to 0.
+     *
+     * @param {import('./webgpu-buffer.js').WebgpuBuffer} storageBuffer - The storage buffer.
+     * @param {number} [offset] - The offset of data to clear. Defaults to 0.
+     * @param {number} [size] - The size of data to clear. Defaults to the full size of the buffer.
+     * @ignore
+     */
+    clearStorageBuffer(storageBuffer, offset = 0, size = storageBuffer.byteSize) {
+
+        // use existing or create new encoder
+        const commandEncoder = this.commandEncoder ?? this.wgpu.createCommandEncoder();
+
+        commandEncoder.clearBuffer(storageBuffer.buffer, offset, size);
+
+        // if we created the encoder
+        if (!this.commandEncoder) {
+            DebugHelper.setLabel(commandEncoder, 'ReadStorageBuffer-Encoder');
+            const cb = commandEncoder.finish();
+            DebugHelper.setLabel(cb, 'ReadStorageBuffer-CommandBuffer');
+            this.addCommandBuffer(cb);
```

Comment:
Why not just `this.wgpu.queue.submit([cb]);`. This doesn't need to happen in sync with the normal frame rate?

Reply:
Well that depends, but there are situation where this needs to happen in sync, so this is the more generic behaviour. As mentioned elsewhere, we can relax it later in some cases.